packages/orbit-components/src/Seat/__tests__/index.test.tsx

```diff
@@ -29,18 +19,16 @@ describe("Seat", () => {
         onClick={onClick}
         description="bur"
         dataTest={dataTest}
+        id="ID"
       />,
     );
 
     await user.click(screen.getByTestId(dataTest));
     expect(onClick).toHaveBeenCalled();
-    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    const el = screen.getByTestId(dataTest);
+    expect(el).toBeInTheDocument();
+    expect(el).toHaveAttribute("id", "ID");
     expect(screen.getByTitle("kek")).toBeInTheDocument();
     expect(screen.getByText("bur")).toBeInTheDocument();
   });
-
-  it.each(Object.values(SIZE_OPTIONS))("it should have size %s", size => {
-    render(<Seat size={size} dataTest="kek" />);
-    expect(screen.getByTestId("kek")).toHaveStyle({ width: width[size], height: height[size] });
```

Comment:
I’ve removed this since .toHaveStyle currently doesn’t work with tailwind and it’s covered by visual test anyway. But if you want to keep it let me know.